client/src/pages/Admins/AdminFinance/UserSelector/UserSelector.js

```diff
@@ -113,6 +116,7 @@ class UserSelector extends React.Component {
 
     componentDidMount () {
         this.getUserList();
+        document.getElementById('selectedUser').focus();
```

Comment:
Direct DOM manipulation is not the preferred way in case of React I believe.
It works well here I am sure, but please consider a more React way to set focus.

Reply:
Indeed. I have refactored the component into a functional component and i am using a reference to the input value instead of a DOM lookup.